src/pages/ItemPage/components/ItemDetailSection.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import ItemCard "./ItemCard";
+
+
+
+function ItemDetailSection() {
+    return(
+        <div>
+            {ItemCard}
```

Comment:
아마도 이렇게 표현하시려고 했던거같은데, ItemDetailSection 안에서 api 요청하는 코드를 아직 넣지 못하신거같네요. 팀프로젝트에서는, '컴포넌트 안에서 API 요청하고 상태로 저장한 후 UI로 표현하기'를 꼭 시도해보시고 완성하시길 바랍니다
```suggestion
            <ItemCard item={item}/>
```